Assets/Scripts/Weapon/Projectile/Projectile.cs

```diff
@@ -18,7 +19,9 @@ public class Projectile : MonoBehaviour
         /**
          * Invoked on hitting a player
          */
-        public event UnityAction<PlayerController, PlayerController> OnHit = delegate { };
+        // TODO: make <PlayerController, PlayerController> a struct with damage/attack detail
```

Comment:
@Teru5244 remember this TODO, need to refine it sometime in the future

Reply:
Got it